From 067daee397765ac4888a9ac64b35b3102210c2f5 Mon Sep 17 00:00:00 2001
From: JasonHuang <Jason.Huang-EE@infineon.com>
Date: Mon, 18 Jul 2022 03:19:24 -0500
Subject: [PATCH 131/179] Remove WARN_ON, while no 6g bw_cap

Remove kernel warning, due to the 6g bw not common currently.

Signed-off-by: JasonHuang <Jason.Huang-EE@infineon.com>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
index dcbfcbd4f914..d211a76788b9 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
@@ -7810,7 +7810,7 @@ static void brcmf_get_bwcap(struct brcmf_if *ifp, u32 bw_cap[])
 				bw_cap[NL80211_BAND_6GHZ] = band;
 				return;
 			}
-			WARN_ON(1);
+			return;
 		}
 		WARN_ON(1);
 		return;
-- 
2.17.1

